spec/monitoring_spec.ts

```diff
@@ -403,20 +402,24 @@ describe('monitoring', () => {
             .perform();
         authoring.save();
         authoring.close();
+        el(['content-profile-dropdown']).click();
+        var items = els(['content-profiles']);
 
-        contentProfiles.openContentProfileDropdown();
-        expect(contentProfiles.totalProfiles().count()).toBe(2);
-        contentProfiles.selectContentProfile();
-        expect(monitoring.getGroupItems(0).count()).toBe(1);
+        browser.wait(ECE.hasElementCount(items, 2));
+        items.get(1).click();
         expect(monitoring.getTextItemBySlugline(0, 0)).toBe('STORY1 SLUGLINE');
-        contentProfiles.removeFilter();
+
+        el(['remove-filter']).click();
         expect(monitoring.getTextItemBySlugline(0, 0)).toBe('STORY1 SLUGLINE');
         expect(monitoring.getTextItem(2, 0)).toBe('item3');
         expect(monitoring.getTextItem(4, 0)).toBe('item4');
-        contentProfiles.openContentProfileDropdown();
-        expect(contentProfiles.totalProfiles().count()).toBe(2);
-        contentProfiles.selectContentProfile();
-        contentProfiles.clearFilters();
+
+        el(['content-profile-dropdown']).click();
+        browser.wait(ECE.hasElementCount(items, 2));
+        items.get(1).click();
```

Comment:
Are you selecting the same item as on line 409? It'd be clearer if you selected by button text. like

`el(['content-profiles'], by.buttonText('content profile 1')).click()`